### PR TITLE
Add ability to message a PR that it broke vagrant devstack

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

File: tubular/github_api.py [hunk 1]
```diff
@@ -26,9 +26,11 @@
...
 
 PR_ON_STAGE_BASE_MESSAGE = '**EdX Release Notice**: This PR has been deployed to the staging environment '
-PR_ON_STAGE_DATE_MESSAGE = 'in preparation for a release to production on {date:%A, %B %d, %Y}.'
-PR_ON_PROD_MESSAGE = '**EdX Release Notice**: This PR has been deployed to the production environment.'
-PR_RELEASE_CANCELED_MESSAGE = '**EdX Release Notice**: This PR has been rolled back from the production environment.'
+PR_ON_STAGE_DATE_MESSAGE = 'in preparation for a release to production on {date:%A, %B %d, %Y}. {extra_text}'
+PR_ON_PROD_MESSAGE = '**EdX Release Notice**: This PR has been deployed to the production environment. {extra_text}'
+PR_RELEASE_CANCELED_MESSAGE = '**EdX Release Notice**: This PR has been rolled back from the production environment. ' \
+                              '{extra_text}'
+PR_BROKE_VAGRANT_DEVSTACK_MESSAGE = '**EdX Release Notice** This PR may have broken devstack. {extra_text}'
```
Comment (doctoryes) [inline tubular/github_api.py]: Seems like `**EdX Release Notice**` should be defined separately.

Comment (doctoryes) [inline tubular/github_api.py]: With the new Docker devstack emerging, perhaps this should read `Vagrant devstack CI`?

File: tubular/tests/test_github.py [hunk 2]
```diff
@@ -401,30 +402,29 @@ def test_message_pr_deployed_stage_weekend(self, message_date, deploy_date):
...
-                github_api.PR_ON_PROD_MESSAGE,
-                False
-            )
-
-    def test_message_pr_release_canceled(self):
+    @ddt.data(
+        (1, github_api.PR_ON_PROD_MESSAGE, '', False, 'message_pr_deployed_prod'),
+        (1337, github_api.PR_ON_PROD_MESSAGE, 'some extra words', False, 'message_pr_deployed_prod'),
+        (867, github_api.PR_RELEASE_CANCELED_MESSAGE, '', True, 'message_pr_release_canceled'),
+        (5, github_api.PR_RELEASE_CANCELED_MESSAGE, 'Elmo does not approve', False, 'message_pr_release_canceled'),
```
Comment (doctoryes) [inline tubular/tests/test_github.py:415]: http://gph.is/1DnqQZY

**Comment by macdiesel** [comment]
@doctoryes comments have been addressed.

**Review by doctoryes — APPROVED** [review]

**Pull request merged**